Month1/Week2/Section1/Exercise3/Main3.cpp

```diff
@@ -0,0 +1,167 @@
+
+/*Exercise 3 - Shapes and Areas
+Create two classes: a Circle class and a Square class. For each class, take in the necessary information to calculate its area and perimeter.  Remember to validate input.
+
+Ask the user if they want to create a circle or a square. For their pick, ask the necessary information to create the shape. Then, tell them the area and the perimeter of their shape.
+
+Your class should have the following methods:
+ExampleShape();
+ExampleShape(/* Params );
+float GetArea();
+float GetPerimeter();
+
+Welcome to create-a-shape, what type of shape do you want?
+1 - Circle
+2 - Square
+Your choice: 1
+
+Enter radius for your circle: 30
+Your shape�s area: 2827.43
+Your shape�s perimeter: 188.5
+
+*/
+
+
+#define _USE_MATH_DEFINES
+#include <iostream>
+#include<string>
+#include <cmath>
+
+class Square
+{
+private:
+	float Side;
+	float AreaSquare;
```

Comment:
This is a descriptive name, but redundant. `Area` would already imply that it's the Square's area since it's inside the Square class. There is no need to have all variables specify that they are part of the square.

Reply:
Done